CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+#### v2.0.0
+
+* `App`
+  * The `App` now extends `Marionette.Application`
+    * **Breaking Changes:**
+      * Move `initState` from `App` constructor to `start`
+      * Remove `restartState` functionality
+    * Add `getInitState` functionality to allow user to override and modify state on App initialization
```

Comment:
not App initialization.  it is before the state is initialized when the app is started.